src/commandlineflags.cc

```diff
@@ -214,9 +210,15 @@ bool IsFlag(const char* str, const char* flag) {
 }
 
 bool IsTruthyFlagValue(const std::string& value) {
-  if (value.empty()) return true;
-  char ch = value[0];
-  return isalnum(ch) &&
-         !(ch == '0' || ch == 'f' || ch == 'F' || ch == 'n' || ch == 'N');
+  if (value.size() == 1) {
+    char v = value[0];
+    return isalnum(v) &&
+           !(v == '0' || v == 'f' || v == 'F' || v == 'n' || v == 'N');
+  } else if (!value.empty()) {
+    return !(value == "false" || value == "False" || value == "FALSE" ||
```

Comment:
what about fAlSe? :)

maybe transform value to lower case before testing?

Reply:
I'd suggest also accepting `off`